pkgs/by-name/ra/rainfrog/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+}:
+let
+  version = "0.2.3";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "rainfrog";
```

Comment:
```suggestion
rustPlatform.buildRustPackage rec {
  pname = "rainfrog";
  version = "0.2.3";
```

Reply:
Unneeded.

Reply:
I don't want to nitpick here, but this is not standard practice in nixpkgs.
I think that if you don't want to use `rec`, the right pattern is `finalAttrs`.
Consistency across the code base is quite valuable according to me.

Feel free to ignore though. This is not a blocker.

Reply:
This is very true.

Reply:
FWIW it’s actually quite common to do this especially for builders that don’t support `finalAttrs`, and the semantics are quite different.